CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 2.7.0
+
+* Elasticsearch: **BC** Paginator now receives the denormalization context to support denormalizing documents using serialization groups.
```

Comment:
Why is it a BC break? This change looks backward compatible (the `Paginator` class is `final` so adding an optional argument is allowed).

Reply:
I'm not totally sure but I think it may be a BC Break because for now the context is not given. This means, when documents are deserialized to objects, it's currently done without any serialization groups. Passing the context that includes groups now, it may affect some existing applications.

Reply:
It makes sense. Maybe should we add this note on the changelog then.

Reply:
`CHANGELOG` has been updated